db/migrate/20231223040257_create_applied_slas.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAppliedSlas < ActiveRecord::Migration[7.0]
+  def change
+    create_table :applied_slas do |t|
+      t.references :account, null: false, foreign_key: true
```

Comment:
lets remove the foreign keys 

ref: https://stackoverflow.com/a/2373517/939299 

> TLDR:  Rails doesn't prevent you from using foreign key constraints in your database, it just doesn't give them to you by default. The "Rails Way" is to depend on Rails to manage your database, cascade your deletes, enforce referential integrity, etc. 

We follow the rails way mostly as the foreign keys make it a little bit more tricky for deletes etc.

Reply:
cool cool ✅